Tensile/KernelWriter.py

```diff
@@ -104,8 +104,32 @@ def makeSchedule(self, kernel, tensorParametersA, tensorParametersB, localWriteE
     lastLoadIter = 0
     if kernel["EnableMatrixInstruction"] and kernel["ScheduleIterAlg"] == 3:
       numMfmaPerIter = kernel["MIWaveTile"][0] * kernel["MIWaveTile"][1] * kernel["InnerUnroll"]
+      # Can locally overrid these ######
+      # number of mfma between last localWrite and barrier
+      numMfmaBetweenLWandBarrier = 1 if kernel["MatrixInstM"] == 32 else 2
+      # number of global read instructions between 2 mfma
+      self.numGlobalReadInsPerMfma = 2 if kernel["MatrixInstM"] == 32 and not kernel["ProblemType"]["TLUA"] and not kernel["ProblemType"]["TLUB"] and kernel["TransposeLDS"] else 1
```

Comment:
may be use latency equation to determine number of global reads. the intention will fail in next generation ..